openmdao/drivers/test/test_pyoptsparse.py

```diff
@@ -131,8 +130,8 @@ def test_simple_paraboloid_equality(self):
         prob.run()
 
         # Minimum should be at (7.166667, -7.833334)
-        assert_rel_error(self, prob['x'], 7.16667, 1e-6)
-        assert_rel_error(self, prob['y'], -7.833334, 1e-6)
+        assert_rel_error(self, prob['x'], 7.16667, 1e-3)
```

Comment:
I think you mean this:

        defOpts = {
            # SLSQP Options
            'ACC': [float, 1e-6],         # Convergence Accurancy

What value would you suggest?